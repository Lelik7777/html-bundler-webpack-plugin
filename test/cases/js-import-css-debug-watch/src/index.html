<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <link s-href="@images/favicons/favicon.svg" rel="icon">
  <link s-href="./style.css" rel="stylesheet">
  <script s-src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Home a1234</h1>
  <p>Body text</p>
  <div class="main">apple</div>
</body>
</html>