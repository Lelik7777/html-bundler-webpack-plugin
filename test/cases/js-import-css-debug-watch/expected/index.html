<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="assets/css/style.2c307bc1.css" rel="stylesheet">
  <style>.inline {
  color: plum;
}
</style>
  <link href="assets/css/app.f6258459.css" rel="stylesheet">
<script src="assets/js/app.ffec14fb.js" defer="defer"></script>
</head>
<body>
  <div class="inline">inline.css - inlined in HTML</div>
  <div class="style">style.css - linked in HTML</div>
  <div class="app1">app1.css - imported in JS</div>
  <div class="app2">app2.css - imported in JS</div>
</body>
</html>